packages/warehouses/src/warehouseClients/PostgresWarehouseClient.ts

```diff
@@ -324,6 +324,24 @@ export class PostgresClient<
             WHERE table_catalog IN (${Array.from(databases)})
               AND table_schema IN (${Array.from(schemas)})
               AND table_name IN (${Array.from(tables)})
+
+            UNION ALL
```

Comment:
I'm trying to test this locally. 

stg_customers was supposed to be a materialized view on jaffle shop, and I can see the columns without this new code, is that expected ? Is there any other thing I need to do to ?

![image](https://github.com/user-attachments/assets/22521898-a58d-4f95-8023-6744ce284912)